### PR TITLE
[hotfix][doc]Fixed an issue where clicking Window Assigner and Window…

### PR DESCRIPTION
**Pull request by meng1222**
## What is the purpose of the change
When I was reading the Windows document, I found that clicking Window Assigner and Window Function in the red circle in the figure could not jump to the correct position, and then I tried to fix it.
image

## Brief change log

Modify Window Assigner and Window Function to the correct title

## Verifying this change

Please make sure both new and modified tests in this PR follows the conventions defined in our code quality guide: https://flink.apache.org/contributing/code-style-and-quality-common.html#testing

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / no) no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / no) no
  - The serializers: (yes / no / don't know) no
  - The runtime per-record code paths (performance sensitive): (yes / no / don't know) no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (yes / no / don't know) no
  - The S3 file system connector: (yes / no / don't know) no

## Documentation

  - Does this pull request introduce a new feature? (yes / no) no
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented) no

### CODE REVIEW

**Review by MartijnVisser — APPROVED** [review]
Thanks for your PR! It would be great if you could also create a PR that fixes this for `master` since this currently only fixes it for our `release-1.14` branch. Is that something that you could do?

**Comment by MartijnVisser** [comment]
@fapaul Could you have a look if this can be merged first? I don't know if it's required that we first merge into `master` before merging into `release-1.14`, let me know what you think.

**Review by MartijnVisser — CHANGES_REQUESTED** [review]
I just noticed one thing, this is also broken for `docs/content.zh/docs/dev/datastream/operators/windows.md`

Could you include that fix in this PR too?

**Comment by meng1222** [comment]
> I just noticed one thing, this is also broken for `docs/content.zh/docs/dev/datastream/operators/windows.md`
> 
> Could you include that fix in this PR too?
I did not notice this problem in the Chinese document, I will commit this problem again, thanks for the reply.

**Comment by meng1222** [comment]
@MartijnVisser I just added a modification to the Chinese document. Could you please see if there are any other problems?

**Review by MartijnVisser — APPROVED** [review]
Thanks a lot for this! The only open thing that's left before merging is to get feedback from @fapaul on https://github.com/apache/flink/pull/18214#issuecomment-1002639637

**Review by fapaul — APPROVED** [review]
@meng1222 thanks for the fix I verified that the links now work. Can you also open a PR to fix this on the master branch?

**Comment by meng1222** [comment]
> @meng1222 thanks for the fix I verified that the links now work. Can you also open a PR to fix this on the master branch?

yes,this is the pr for the master, can you merge it please?

**Comment by meng1222** [comment]
> > @meng1222 thanks for the fix I verified that the links now work. Can you also open a PR to fix this on the master branch?
> 
> yes,this is the pr for the master, can you merge it please?

https://github.com/apache/flink/pull/18242/

**Pull request merged**
